lib/pubsub/subscription.js

```diff
@@ -401,7 +401,7 @@ Subscription.prototype.pull = function(options, callback) {
  */
 Subscription.prototype.setAckDeadline = function(options, callback) {
   var body = {
-    ackId: options.ackId,
+    ackIds: util.arrayize(options.ackIds),
```

Comment:
Nit:
I think you break the compatibility here, but I don't think it's a big deal.